tests/Games/NexusMods.Games.TestFramework/AIsolatedGameTest.cs

```diff
@@ -377,6 +379,49 @@ private AbsolutePath GetArchivePath(Hash hash)
         store.TryGetLocation(Connection.Db, hash, null, out var archivePath, out _).Should().BeTrue("Archive should exist");
         return archivePath;
     }
+    
+            /// <summary>
+    /// Uses Verify to validate all the states of the game, previously applied states, etc.
+    /// </summary>
+    protected void LogDiskState(StringBuilder sb, string sectionName, string comments = "", Loadout.ReadOnly[]? loadouts = null)
```

Comment:
Somewhat misleading description, this isn't using Verify at all, this method just prints the Loadouts as tables into the string builder.